test/state-manager_test.html

```diff
@@ -17,6 +17,8 @@
       <template>
         <video-player
         configuration='{
+        "quizValidationCallback": "",
+        "quizQuestions" : [],
```

Comment:
This needs to be adapted to the new nested structure. However, it could also be removed at all as there are no tests for the quizzes, right?

Reply:
Ah yeah, forgot about these. Thanks for spotting :)